### ISSUE TITLE
Add Qwen2.5-VL-7B-Instruct support

### ISSUE THREAD

**Issue by LarsLiden** [LarsLiden/EmbodiedBench#8]
In PR #7 you added support for AzureOpenAI.  I now want you to do the same for Qwen2.5-VL-7B-Instruct.  I've provided an example below that you should follow closely to impliment this. You may need to alter the call signature of "get_response" to work with this repository

To select the model please allow a new option like:

python -m embodiedbench.main env=eb-hab model_name={path to local checkpoint} model_type=qwen_instruct exp_name='baseline' tp=1
=========================

from transformers import Qwen2_5_VLForConditionalGeneration, AutoProcessor
from qwen_vl_utils import process_vision_info
import torch
import sys

class QwenVLActor:
    def __init__(self, model_path: str, temperature: float):

        local_files_only = model_path != "Qwen/Qwen2.5-VL-7B-Instruct"
        self.model = Qwen2_5_VLForConditionalGeneration.from_pretrained(
            model_path, torch_dtype=torch.float16, device_map="auto", local_files_only=local_files_only
        )
        self.processor = AutoProcessor.from_pretrained(model_path)
        self.temperature = temperature

    def get_response(self, image_path: str, prompt: str) -> str:
        try:
            messages = [
                {
                    "role": "user",
                    "content": [
                        {
                            "type": "image",
                            "image": image_path,
                        },
                        {"type": "text", "text": prompt},
                    ],
                }
            ]

            text = self.processor.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
            image_inputs, video_inputs = process_vision_info(messages)

            inputs = self.processor(
                text=[text],
                images=image_inputs,
                videos=video_inputs,
                padding=True,
                return_tensors="pt",
            )
            inputs = inputs.to("cuda")
            generated_ids = self.model.generate(
                **inputs,
                max_new_tokens=1500,
                temperature=self.temperature,
                do_sample=True if self.temperature > 0 else False,
            )
            generated_ids_trimmed = [out_ids[len(in_ids) :] for in_ids, out_ids in zip(inputs.input_ids, generated_ids)]
            output_text = self.processor.batch_decode(
                generated_ids_trimmed, skip_special_tokens=True, clean_up_tokenization_spaces=False
            )[0]

            return output_text.strip()
        except Exception as e:
            print(f"--= !!! Invalid QWEN Request: {e} !!! =--", file=sys.stderr)
            sys.exit(1)
